<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SIGNUP</class>
 <widget class="QDialog" name="SIGNUP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SIGN UP</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#SIGNUP {
background-image:url(:/ho.jpg);
}

QFrame{
background-color: rgba(200, 200, 200, 0.5); 
border-radius: 20px;
}

#frame_2{
background-image: url(:/logo_ig.png);
background-color: transparent;
}

#createButton {
background-color: #3399cc;
border-radius: 50px;
color: white;
}

Line {
background-color: #696969;
}

QToolButton{
background:#8B0000;
border-radius:60px;

}
QLabel{
background:transparent;
color:#696969;
}

#label_4{
color:white;
font-family: forte;
font-size: 40px;
}
QLineEdit{
background:transparent;
border:none;
color:#696969;
border-bottom: 1px solid #717072;
}


</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>531</width>
     <height>561</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="usernameEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>351</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>351</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>351</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="phoneEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>351</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Phone</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dobEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>351</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Date of Birth</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="genderEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>351</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Gender</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="privacyEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>380</y>
      <width>351</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Private/Public</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>430</y>
      <width>351</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sign up to see new posts from your friends</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>251</width>
      <height>99</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
